loader_hub/googledrive/README.md

```diff
@@ -0,0 +1,38 @@
+# Google Drive Loader
+
+This loader currently reads files from google drive using folder/ file ids.
+
+To use this loader, you need to pass in a List of file id's or folder id.
+
+### folder_id
+You can extract a folder_id directly from its drive URL. 
+
+For example, the folder_id of https://drive.google.com/drive/folders/1w7XryYu6mL9VLmfyqUkA4_fRnDbsCqV- is 1w7XryYu6mL9VLmfyqUkA4_fRnDbsCqV-
+
+### file_id
+You can extract a file_id directly from its sharable drive URL. 
+
+For example, the file_id of https://drive.google.com/file/d/1LEqD_zQiOizKrBKZYKJtER_h6i49wE-y/view?usp=sharing is 1LEqD_zQiOizKrBKZYKJtER_h6i49wE-y
+
+## Usage
+
+We need `credentials.json` and `client_secrets.json` files to use this reader.
```

Comment:
In addition to credentials, I also had to enable Google Drive API in my Google API Console for the app corresponding to these credentials. Let's make a note of that too?

Reply:
Yes, sure. Will do that.